src/components/renderer/KopytkoRoot.brs

```diff
@@ -0,0 +1,18 @@
+sub initKopytkoRoot(dynamicProps as Object)
+  for each prop in dynamicProps
+    m.top.observeFieldScoped(prop, "KopytkoRoot_dynamicPropChanged")
+  end for
+
+  dynamicPropsValues = {}
+  for each prop in dynamicProps
+    dynamicPropsValues[prop] = m.top[prop]
+  end for
+
+  initKopytko(dynamicPropsValues)
+end sub
+
```

Comment:
Shouldn't there be also a `destoryKopytkoRoot`, which `unobserveFieldScoped` and `destroyKopytko`?

Reply:
That's a good call. I think it's good to have one for consistency to clear everything what `initKopytkoRoot` adds. 

On the other hand, we potentially don't need to unobserve these fields, and in the currently suggested approach, with observing each field manually, an unobserve is not called anywhere. E.g. for the item component in the RowList, the RowList controls the removal of items, so either `initKopytko` and unobserves are not called. 

Anyway, having such destroy function will allow removing observers in case it's ever needed.